docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+
+   you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/Vagrantfile && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/cluster.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.crt && \
+   curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/user.key && \
+   vagrant up && \
+   [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+   ssh-add ~/.vagrant.d/insecure_private_key
+
+Option B: Installing Flocker on AWS or Rackspace
+************************************************
+
+The two instances of Flocker each run on a separate host.
+Flocker manages the links, ports, and volumes associated with Docker containers and can move them around after deployment.
+To install Flocker (plus dependencies), follow the links to the direct instructions:
+
+- :ref:`AWS install instructions <aws-install>`
+- :ref:`Rackspace install instructions <rackspace-install>`
+
+Step 3: Deploying an app on the first host
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You will now have the Flocker client installed on your local machine and two instances of Flocker, each on a different host.
+Now you will create two Docker containers on one of the hosts.
+One is a Python web application and the other is Redis server, which stores its data on a volume.
+
+Download the first 2 .yml files that we have provided:
+
+.. version-code-block:: console
+
+	you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/docker-compose.yml
+	you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+
+.. note:: There are 3 .yml files to download. These contain the application and deployment configuration. You can edit these files if you need to change the IP addresses to match your hosts'.
+
+The ``docker-compose.yml`` file describes your distributed application (note, Docker Compose was formerly known as Fig):
+
+    .. literalinclude:: docker-compose.yml
+       :language: yaml
+
+The ``deployment-node1.yml`` file describes which containers to deploy, and where:
+
+    .. literalinclude:: deployment-node1.yml
+       :language: yaml
+
+.. note:: If you are using real servers on AWS, you'll need to change the IP addresses in the deployment file.
+
+Secondly, install the web application and server on the first host:
+
+.. prompt:: bash [you@laptop:~$]
+
+	flocker-deploy 172.16.255.250 deployment-node1.yml fig.yml
+
+Visit http://172.16.255.250/ (or the IP of the first host that you are using). You will see the visit count displayed.
+
+Visit http://172.16.255.251/ (or the IP of the second host that you are using).
+You will see that the count persists because Flocker routes the traffic from either host named in the deployment file to the one that has the application.
+
+Run the following from within the :file:`/vagrant-flocker` folder to check that the Redis server container is running on the first host:
```

Comment:
There is no `vagrant-flocker` folder because the Vagrantfile is curl-ed not cloned with Git. I think that this guide should be run through at least once with all permutations.